### ISSUE TITLE
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference

### ISSUE THREAD

**Issue by hissingpanda** [RefugeRestrooms/refugerestrooms-android#26]
App crash as of v1.3. Stack Trace is below.

````
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
	at org.refugerestrooms.views.MainActivity.loadBathrooms(MainActivity.java:900)
	at org.refugerestrooms.views.MainActivity.onSearchResults(MainActivity.java:738)
	at org.refugerestrooms.servers.Server$2.onResponse(Server.java:51)
	at org.refugerestrooms.servers.Server$2.onResponse(Server.java:42)
	at com.android.volley.toolbox.JsonRequest.deliverResponse(JsonRequest.java:65)
	at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:99)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:158)
	at android.app.ActivityThread.main(ActivityThread.java:7224)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```

**Comment by hissingpanda** [comment]
Fixed as of #30

**Comment by sam0697** [comment]
boolean isMain;
 if(null != grades.getIsMian()){
                       IsMian=grades.getIsMian();
    
                    }
